src/java.base/share/classes/javax/net/ssl/SSLSessionContext.java

```diff
@@ -106,7 +106,7 @@ public interface SSLSessionContext {
      *
      * @see #getSessionTimeout
      */
-    public void setSessionTimeout(int seconds)
+    void setSessionTimeout(int seconds)
                  throws IllegalArgumentException;
```

Comment:
IllegalArgumentException is a runtime exception, it's unusual to have "throws IllegalArgumentException". It would not impact compatibility to drop it. The important thing is that the javadoc has the `@throws` describing when it is thrown.

Reply:
I'll drop it then. Thanks for noticing this.